test/xpu_api/utilities/utility/forward/move.pass.cpp

```diff
@@ -0,0 +1,149 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/utility>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+struct move_only
+{
+    move_only(const move_only&) = delete;
+    move_only& operator=(const move_only&) = delete;
+
+    move_only() = default;
+    move_only(move_only&&) = default;
+    move_only&
+    operator=(move_only&&)
+    {
+        return *this;
+    }
+};

```

Comment:
It seems like `MoveOnly` type you introduced as part of this PR can be used instead.

Reply:
Done